sdk/rust/oak/src/io/mod.rs

```diff
@@ -115,6 +116,7 @@ fn send_init<Init: Encodable + Decodable, Command: Encodable + Decodable>(
     sender: Sender<oak_io::InitWrapper<Init, Command>>,
     init: Init,
     command_channel_label: &Label,
+    keep_sender_open: bool,
```

Comment:
If we do need this param, could you create a dedicated self-documenting enum? Seeing `true` and `false` at each call place is not particularly self-explanatory.

Reply:
Removed. It was part of my experimentation :)